packages/xrpl/src/models/transactions/AMMDeposit.ts

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable complexity -- required for validateAMMDeposit */
```

Comment:
This doesn't need to happen in this PR, but perhaps this should be turned off for the `validate` functions

Reply:
I created a [jira ticket](https://ripplelabs.atlassian.net/browse/DGE-2043) to track this later.

Reply:
Created an issue to track this later - https://github.com/XRPLF/xrpl.js/issues/2407